packages/app-store/_utils/useAddAppMutation.ts

```diff
@@ -12,12 +12,14 @@ function useAddAppMutation(type: App["type"], options?: Parameters<typeof useMut
     };
     const stateStr = encodeURIComponent(JSON.stringify(state));
     const searchParams = `?state=${stateStr}`;
+
     const res = await fetch(`/api/integrations/${appName}/add` + searchParams);
+
     if (!res.ok) {
       throw new Error("Something went wrong");
     }
-    const json = await res.json();
-    window.location.href = json.url;
+
```

Comment:
I was having some errors while doing res.json() saying the body wasn't parsing correctly.

Reply:
Does this breaks for the other apps?

Reply:
It didn't break on my end, and it shouldn't broke for other apps.
@joeauyeung Can you help me test this one?